stylesheet.css

```diff
@@ -1,195 +1,742 @@
+#open li {
+  border-bottom-width: medium;
+  border-bottom: 0.1rem solid lightgrey;
+  border-color: #8F8F8F;
+}
+
+#open figcaption > header {
+  font-size: 1.75rem;
+  padding-bottom: 0.5rem;
+}
+
+#open figcaption > p {
+  line-height: 1.3;
+  padding-bottom: 0.5rem;
+}
+
+#open li > section {
+  padding: 1rem 0;
   display: flex;
-  -webkit-justify-content: center;
-  -ms-flex-pack: center;
+  flex-direction: row;
+  justify-content: space-around;
+}
+
+#open li > section > img {
+  width: 6rem;
+  height: 6rem;
+}
+
+/* community section */
+#community li {
+  padding: 1.5rem 0;
+}
+
+#community figure {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
   justify-content: center;
-  -webkit-align-items: center;
-  -ms-flex-align: center;
+}
+
+#community figcaption {
+  display: flex;
+  flex-direction: column;
   align-items: center;
 }
 
-/*-- open source section --*/
-.open-logo {
-  display: block;
-  margin: auto;
+#community figcaption > header {
+  padding: 1rem 0;
+  font-size: 1.5rem;
+  font-weight: 300
 }
 
-.open-border {
-  display: -webkit-flex;
-  display: -ms-flex;
-  -webkit-align-items: stretch;
-  -ms-flex-align: stretch;
-  align-items: stretch;
-  border-bottom: 2px solid lightgrey;
+#community figcaption > p {
+  font-size: 0.9rem;
+  font-weight: 100;
+  line-height: 1.6;
+  text-align: center;
 }
 
-.open-part {
+/* sumup-page section*/
+#sumup-page figure > img {
   display: block;
-  margin: auto;
-  float: right;
+  padding: 0 0;
+  height: auto;
+  width: 100%;
 }
 
-/*-- comm section --*/
-.comm-logo {
+#sumup-page section > section {
   display: block;
-  margin: auto;
-  -webkit-flex: 1 0 0;
-  flex: 1 0 0;
+  line-height: 1.5;
+  text-align: center;
+  padding: 4rem 2rem;
 }
 
-/*-- sum up page --*/
-.address {
-  font-style: normal;
+#sumup-page section > section > img {
+  margin: 0.5rem 0;
 }
 
-.sum-contact{
-  border-top-left-radius: 5px;
-  border-top-right-radius: 5px;
-  border-bottom-right-radius: 5px;
-  border-bottom-left-radius: 5px;
-  width: 150px; 
-  height: 25px;
+#sumup-page section > section > p {
+  text-align: left;
+  font-weight: 100;
 }
 
-.sum-main-box {
-  align-items: center;
-  justify-content: center;
+#sumup-page address {
+  font-style: normal;
+  font-weight: 100;
+  text-align: left;
+  padding: 0.5rem 0;
+}
+
+/* sumup-contact section */
+#sumup-contact {
+  padding: 1rem 1.25rem;
   text-align: center;
+  font-family: "Montserrat";
+}
+
+#sumup-contact p {
+  line-height: 1.3;
 }
 
-.sum-green {
+.sumup-slogan {
   color: #73e600;
-  font-size: 150%;
-  font-weight: normal;
+  font-size: 1.3rem;
+  margin: 1.5rem 0;
+  line-height: 1.5;
+  font-weight: 100;
 }
 
-/*-- footer --*/
-.footer {
-  background-color: blue;
+#sumup-contact input {
+  border: 0.02rem solid #DADADA;
+  border-radius: 0.25rem;
+  font-size: 1rem;
+  font-weight: 100;
+  margin: 0.4rem 0;
+  width: 100%;
+  padding: 0.4rem;
+}
+
+#sumup-contact input.sum-up-submitbox {
```

Comment:
I am pretty sure this one looks the same on mobile as the one in intro section. You don't need separate class for it on mobile